src/components/personal-information/index.js

```diff
@@ -194,6 +224,68 @@ const PersonalInfoComponent = ({
+                                        <br/>                            
+                                        <div className={styles.fieldWrapperRadio}>
+                                            <label>Ticket is for:</label>
+                                            <RadioList
+                                                id={`ticket-self-radio`}
+                                                value={ticketOwnerOption}
+                                                options={radioListOptions}
+                                                onChange={handleRadioButtonChange}
+                                                inline
+                                                html
+                                            />
+                                            {ticketOwnerError && 
+                                                <>
+                                                    <br/>
+                                                    <div className={styles.fieldError} data-testid="company-error">This field is required.</div>
+                                                </>
+                                            }
+                                        </div>
+                                        {ticketOwnerOption === 'someoneElse' && 
+                                            <>
+                                                <div className={styles.fieldWrapper}>
+                                                    <div className={styles.inputWrapper}>
+                                                        <input type="text" placeholder="Email *" defaultValue={personalInfo.attendee.email || ''} 
+                                                        {...register("attendee.email", { 
+                                                            validate: {                                                            
+                                                                emailRequired: (value) => {                                                                
+                                                                    const formValues = getValues();
+                                                                    const firstName = formValues.attendee.firstName;
+                                                                    const lastName = formValues.attendee.lastName;
+
+                                                                    if (firstName && lastName && !value) {
+                                                                        return 'This field is required.';
+                                                                    }
+                                                                    
+                                                                    return true;
+                                                            }}, 
+                                                            pattern: /^\S+@\S+$/i 
```

Comment:
this regex does not provide a valid email validation
lets look for a better one like
https://pdw.ex-parrot.com/Mail-RFC822-Address.html